### PR TITLE
Story6task2

### PR DESCRIPTION
**Pull request by Sipho-Ndlovu**
Created the ReviewSummary component for story6...

### CODE REVIEW

File: src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function ReviewSummary ({reviews}) {
+
+    let reviewCount = {reviews}.reviews.length;
```
Comment (Kel-E-creator) [inline src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx]: Please remove semi-colons at the ends of lines 3,4,5,6.  If these are necessary for the code to work, please ignore

File: src/components/BookDetail/BookDetail.css [hunk 2]
```diff
@@ -12,6 +12,9 @@
 .detail-container-text{
```
Comment (Kel-E-creator) [inline src/components/BookDetail/BookDetail.css:14]: Please add a white space between lines 1 & 2 / 11&12 / 13&14 /16 &17 /

File: src/components/BookDetail/BookDetail.jsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { useParams } from "react-router-dom"
 import Reviews from "./Reviews/Reviews";
```
Comment (Kel-E-creator) [inline src/components/BookDetail/BookDetail.jsx]: Please remove semi colon from end of line 3 and 6 and 10 if it is not functional

**Review by Kel-E-creator — CHANGES_REQUESTED** [review]
Please see minor changes to make.  Great job code looks very good :-)

File: src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx (hunk 1 above)
Comment (Sipho-Ndlovu) [reply src/Components/BookDetail/Reviews/ReviewSummary/ReviewSummary.jsx]: Done, semi-colons removed.

File: src/components/BookDetail/BookDetail.css (hunk 2 above)
Comment (Sipho-Ndlovu) [reply src/components/BookDetail/BookDetail.css:14]: Done whitespaces added.

File: src/components/BookDetail/BookDetail.jsx (hunk 3 above)
Comment (Sipho-Ndlovu) [reply src/components/BookDetail/BookDetail.jsx]: Done, Semi-colons removed.

**Pull request merged**